tests/test_discovery_sample/tests_sample.py

```diff
@@ -41,6 +41,32 @@ def test_multiple_tags(self):
         self.assertEqual(1, 1)
 
 
+@tag('foo')
```

Comment:
Perhaps it would be better to put these tests in a new file, e.g. ` tests/test_discovery_sample/tests_tag_inheritance.py` so that existing tests don't have to be changed.

Reply:
With the module you are looking to have created I would still have to technically change the ``test_file_path`` function, given that would still discover those new tests. Are you OK with that change or would you want the new tests to be in a new package altogether then?

Reply:
I think we should first reorganize the test apps as proposed in https://github.com/django/django/pull/9541. Then you can  add the new tests in `tests/test_runner_apps/tagged/tests.py` or in a new flie in that directory.